bc_obps/registration/api/v2/user_operators.py

```diff
@@ -1,16 +1,41 @@
+from django.db.models import QuerySet
+from ninja import Query
 from common.permissions import authorize
 from common.api.utils import get_current_user_guid
 from django.http import HttpRequest
 from registration.decorators import handle_http_errors
+from registration.models import UserOperator
 from registration.schema.generic import Message
 from registration.api.router import router
 from service.error_service.custom_codes_4xx import custom_codes_4xx
-from typing import Literal, Tuple
-
+from typing import Literal, Tuple, Optional, List
 from registration.constants import USER_OPERATOR_TAGS_V2
 from registration.schema.v2.operator import OperatorIn
-from registration.schema.v2.user_operator import UserOperatorOperatorOut
+from registration.schema.v2.user_operator import UserOperatorOperatorOut, UserOperatorListOut, UserOperatorFilterSchema
 from service.user_operator_service_v2 import UserOperatorServiceV2
+from ninja.pagination import paginate, PageNumberPagination
+
+## GET
+@router.get(
+    "/v2/user-operators",
+    response={200: List[UserOperatorListOut], custom_codes_4xx: Message},
+    tags=USER_OPERATOR_TAGS_V2,
+    description="""Retrieves a paginated list of user operators.
+    The endpoint allows authorized IRC roles to view user operators, sorted by various fields such as creation date,
+    user details, and operator legal name.""",
+    auth=authorize("authorized_irc_user"),
+)
+@handle_http_errors()
+@paginate(PageNumberPagination)
+def list_user_operators_v2(
+    request: HttpRequest,
+    filters: UserOperatorFilterSchema = Query(...),
+    sort_field: Optional[str] = "created_at",
```

Comment:
created_at isn't one of the grid columns, does that matter?

Reply:
Yes, `created_at` serves as the default value for sorting when users do not specify their own sorting preferences.(It's used for the query)